remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -83,6 +84,21 @@ public void setup()
         given(nodesMock.validateNode(StoreRef.STORE_REF_WORKSPACE_SPACESSTORE, TAG_ID)).willReturn(TAG_NODE_REF);
         given(taggingServiceMock.getTagName(TAG_NODE_REF)).willReturn(TAG_NAME);
     }
+    @Test
+    public void testGetTags() {
+        final List<String> tagNames = List.of("testTag","tag11");
+        final List<Tag> tagsToCreate = createTags(tagNames);
+        given(taggingServiceMock.createTags(any(), any())).willAnswer(invocation -> createTagAndNodeRefPairs(invocation.getArgument(1)));
+        given(parametersMock.getInclude()).willReturn(List.of("count"));
+        final List<Tag> actualCreatedTags = objectUnderTest.createTags(tagsToCreate, parametersMock);
+        final List<Tag> expectedTags = createTagsWithNodeRefs(tagNames).stream()
+            .peek(tag -> tag.setCount(0))
+            .collect(Collectors.toList());
+        assertThat(actualCreatedTags)
+            .isNotNull()
+            .isEqualTo(expectedTags);
+        assertEquals(expectedTags, actualCreatedTags);
```

Comment:
I think you can reduce these four lines down to one. The final assertion covers everything above it.

Reply:
Incorporated